src/App/Mail/SendTodoDeadlineExpiredMailSubscriber.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace Prooph\ProophessorDo\App\Mail;
+
+use Prooph\ProophessorDo\Model\Todo\Event\TodoWasMarkedAsExpired;
+use Prooph\ProophessorDo\Projection\Todo\TodoFinder;
+use Prooph\ProophessorDo\Projection\User\UserFinder;
+use Zend\Expressive\Template\TemplateRendererInterface;
+use Zend\Mail\Transport\TransportInterface;
+use Zend\Mail;
+
+/**
+ * Class SendTodoDeadlineExpiredMailSubscriber
+ *
+ * @package Prooph\ProophessorDo\App\Mail
+ * @author Michał Żukowski <michal@durooil.com
+ */
+final class SendTodoDeadlineExpiredMailSubscriber
+{
+    private $userFinder;
+    private $todoFinder;
+    private $mailer;
+
+    public function __construct(
+        UserFinder $userFinder,
+        TodoFinder $todoFinder,
+        TransportInterface $mailer
+    ) {
+        $this->userFinder = $userFinder;
+        $this->todoFinder = $todoFinder;
+        $this->mailer = $mailer;
+    }
+
+    public function __invoke(TodoWasMarkedAsExpired $event)
+    {
+        $todo = $this->todoFinder->findById($event->todoId()->toString());
+        $user = $this->userFinder->findById($todo->assignee_id);
+
+        $message = sprintf(
+            'Hi %s! Just a heads up: your todo `%s` has expired on %s.',
+            $user->name,
+            $todo->text,
+            $todo->deadline
+        );
+
+        $mail = new Mail\Message();
```

Comment:
@prolic what about existing `SendTodoReminderMailSubscriber` class? it does not go through the CommandBus